dev/com.ibm.ws.security.javaeesec.cdi/resources/com/ibm/ws/security/javaeesec/cdi/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -17,50 +17,45 @@
+JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED.useraction=Make sure that identity stores are configured or that the user registry uses a supported credential class to authenticate the user.
 
-JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED=CWWKS1927E: The supplied credential class {0} is not supported by the user registry. Use either javax.security.enterprise.credential.UsernamePasswordCredential or javax.security.enterprise.credential.BasicAuthenticationCredential for authentication.
-JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED.explanation=The supplied credential class for the authentication is not supported by the user registry. Use either javax.security.enterprise.credential.UsernamePasswordCredential or javax.security.enterprise.credential.BasicAuthenticationCredential for authentication.
-JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED.useraction=Make sure that the supported credential class is used for authentication by using the user registry.
+JAVAEESEC_CDI_INFO_NO_IDENTITY_STORE=CWWKS1930I: The configured IdentityStore object was not found. If a user registry is configured, it will be used instead. If the IdentityStore object must be used, make sure that the IdentityStore object is configured properly.
+JAVAEESEC_CDI_INFO_NO_IDENTITY_STORE.explanation=The reason that the IdentityStore object was not found varies. The error messages for the IdentityStore object provide information about why the IdentityStore object was not found.
+JAVAEESEC_CDI_INFO_NO_IDENTITY_STORE.useraction=Investigate any error messages from the IdentityStore object, and make corrections based on the error messages.
 
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE=CWWKS1930W: The configured IdentityStore object was not found. If a user registry is configured, it will be used instead. Make sure that the IdentityStore object configured by the application is available.
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.explanation=The reason that the IdentityStore object was not found varies. The error messages for the IdentityStore object provide information about why the IdentityStore object was not found.
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.useraction=Investigate any error messages from the IdentityStore object, and make corrections based on the error messages.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS=CWWKS1931E: The {1} application  failed deployment because both a login-config element in the web.xml file and the HttpAuthenticationsMechanism element were specified for the {0} module. Make sure that only one authentication  mechanism is configured.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because both the login-config element and the HttpAuthenticationMechanism element are defined. Only one of these authentication mechanisms can be used in your application.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=Make sure that only one of  the authentication mechanisms is configured.
 
```

Comment:
Extra space:
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=Make sure that only one **of the** authentication mechanisms is configured.